doc/benchmark/modules/ROOT/pages/run/running-benchmark-cli.adoc

```diff
@@ -34,7 +34,7 @@ To use a different server URL, realm and scenario:
 
 [source,bash]
 ----
-./kcb.sh --scenario=keycloak.scenario.authentication.AuthorizationCode --server-url=http://localhost:8080 --realm-name=test-realm
+./kcb.sh --scenario=keycloak.scenario.authentication.AuthorizationCode --server-url=http://keycloak.127.0.0.0.nip.io --realm-name=test-realm
```

Comment:
same comment as above, once we add the reference, change this IP to match the dummy IP we provide for the previous command